QATCH/QModel/src/models/v6_yolo/v6_yolo_live.py

```
@@ -732,3 +809,75 @@ def stop(self) -> None:
+    """
+
+    def __init__(self, classifier_ref):
+        """
+        Args:
+            classifier_ref: A reference to the QModelV6YOLO_Live instance
+                            so we can access `_data` directly for plotting.
+        """
+        self.classifier = classifier_ref
+
+        # Setup Figure and Axes
+        self.fig, self.ax1 = plt.subplots(figsize=(10, 6))
```

Ruff (0.15.13)

[error] Undefined name `plt`

(F821)